upgrade/patches/64_custom_field_option_order.sql

```diff
@@ -0,0 +1,6 @@
+/**
+ * Control the order of custom fields
+ *
+ */
+ALTER TABLE {{%custom_field_option}} ADD COLUMN cfo_rank int(10) NOT NULL DEFAULT 0;
```

Comment:
avoid using WIDTH to integer columns, use just `INT`, as it's misleading, also, if you don't need negative values, use `UNSIGNED INT`. and perhaps even `UNSIGNED TINYINT` if 255 values is enough (or `SMALLINT`)

also use `AFTER` or `BEFORE` to order numeric fields first in the table.